### PR TITLE
Update to use go generate per established pattern in infratographer community

### PR DESCRIPTION
**Pull request by cherchezlafemme**
- removes dependency on localhost
- removes deviation from established pattern in other infratographer repos

after this PR is merged, running `go generate ./...` works

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,8 +2,6 @@ module go.infratographer.com/load-balancer-api
 
 go 1.20
 
-replace github.com/testcontainers/testcontainers-go => github.com/testcontainers/testcontainers-go v0.0.0-20230503220718-0df60de8ccd8
```
Comment (tylerauerbeck) [inline go.mod:5]: Don't think we're ready to remove this quite yet per https://github.com/infratographer/load-balancer-api/pull/127#pullrequestreview-1415530704

**Review by tylerauerbeck — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (cherchezlafemme) [reply go.mod]: reverted back to as it was

**Comment by tylerauerbeck** [comment]
@cherchezlafemme Could you take a look at the failing tests?

**Comment by tylerauerbeck** [comment]
🤦🏻 Had to merge main in again and it looks like we've got some more failing tests.

**Review by tylerauerbeck — APPROVED** [review]

**Pull request merged**